projects/distribution/distribution/GIT_TAG

```diff
@@ -0,0 +1 @@
+v2.7.1
```

Comment:
We have received Depedabot alerts regarding Distribution project tags below `v2.8.0`? Can we build off [that tag](https://github.com/distribution/distribution/releases/tag/v2.8.0) instead, considering they've added CVE patches?

Reply:
Is this the same project at docker/distribution? If so, I think it needs to be `v2.8.0-beta.1`

Reply:
Yes it's the same project, but the `beta.1` tag is just a pre-release tag (that's why the release changelogs for `v2.8.0-beta.1` and the actual release tag `v2.8.0` are identical). Also there is only one commit diff between these two, that too in a README. 

https://github.com/distribution/distribution/compare/v2.8.0-beta.1..v2.8.0

I don't think it would hurt to just use the latest, but I may be wrong,

Reply:
Let's get this through to unblock another PR (https://github.com/aws/eks-anywhere-build-tooling/pull/367). I'll make another PR to deal with CVE.